Remove Unix standard settings for Solaris 11.4, so we don't exclude all
features added between XPG2 & XPG7, now that these are exposed by default.

To send upstream, need to adjust to right version for each OS level

--- glib-2.82.0/meson.build.orig
+++ glib-2.82.0/meson.build
@@ -2477,8 +2477,6 @@
 
 # FIXME: How to detect Solaris? https://github.com/mesonbuild/meson/issues/1578
 if host_system == 'sunos'
-  glib_conf.set('_XOPEN_SOURCE_EXTENDED', 1)
-  glib_conf.set('_XOPEN_SOURCE', 2)
   glib_conf.set('__EXTENSIONS__',1)
 endif
 
